scripts/homepage.js

```diff
@@ -0,0 +1,63 @@
+const seeMore = document.querySelector('.seemore');
+const seeArtist = document.querySelector('.artist-section');
+const artsitSection = document.querySelector('.artist-container');
+
+seeMore.addEventListener(('click'), () => {
+  seeArtist.classList.toggle('setoff');
+});
+
+const artistContainer = [
+  {
+    name: 'ELLA',
+    title: 'Main artist',
+    description: 'Answering the call of nostalgia and in a bid to bridge the gap between the new-school gospledance and socia dance.',
+    Image: 'Images/main artiest .png',
+  },
+  {
+    name: 'mr.kah',
+    title: '3th Artist',
+    description: 'Trap Rapper/Songwriter from Bamenda, Cameroon. Currently signed to B.D.A GANG MUSIC TRAP STAR! VOICE OF THE STREETS! CAMEROON!',
+    Image: 'Images/mr kah.png',
+  },
```

Comment:
- Hi, you have done an incredible job and you are almost there. As stated by the previous reviewer, I would advise you to do as much as possible to match the original design. I marked red in the original picture to attract your attention to where you need to improve. 
- NB: Use the exact same color, text, font size, backgrounds, border, etc ...
- Your design:
![Capture](https://github.com/ClarenceAtim/Capstone-project/assets/63972670/de106607-e386-4e18-9afd-a8dfc3abe958)

- Original design:

![Capture1](https://github.com/ClarenceAtim/Capstone-project/assets/63972670/6a50c85b-66f5-4501-897d-c4166c390c4d)

Reply:
i did everything that was require by the previouse reviewer so i dont understand what you are talking about